networks/home/integration/full-node.md

```diff
@@ -2,7 +2,19 @@
 
 ## Spec Requirement
 
-Same as the Kusama full node requirements.
+You can check if your machine satisfy the spec requirement by using the following make command to benchmark your machine.
+
+
+```
+make benchmark-machine
```

Comment:
can we run this using docker?

Reply:
Added docker command. I have trouble running it on my machine, probably due to M1 chip.
Can you double check this? thanks